lib/openstudio-standards/standards/necb/common/btap_data.rb

```diff
@@ -2089,11 +2089,25 @@ def phius_performance_indicators(model)
     tcd_degree_f = OpenStudio.convert(tcd_degree_c.to_f, 'C', 'F').get
 
     ### IGHL (Irradiance, Global, at the heating design condition) (Btu/h.ft2) in PHIUS, 2021
-    solar_irradiance_on_heating_design_day_w_per_m_sq = BTAP::Environment::WeatherFile.new(weather_file).get_ghi_on_heating_design_day
+    average_daily_global_irradiance_w_per_m2_array = []
+    model.getDesignDays.each do |design_day|
+      next unless design_day.dayType == 'WinterDesignDay'
+
+      average_daily_global_irradiance_w_per_m2 = OpenstudioStandards::Weather.design_day_average_global_irradiance(design_day)
+      average_daily_global_irradiance_w_per_m2_array << average_daily_global_irradiance_w_per_m2
+    end
+    solar_irradiance_on_heating_design_day_w_per_m_sq = average_daily_global_irradiance_w_per_m2_array.min
```

Comment:
I did not follow why min (or max) of "average_daily_global_irradiance_w_per_m2_array" is calculated here... wouldn't that be only one day which is WinterDesignDay (or SummerDesignDay)?

Reply:
So what this is saying is for the heating condition - look at all the heating design days, and log the average daily global irradiance.
Then, take the minimum of these values, meaning use the one which has the lowest average daily global irradiance. This is the most extreme condition - heating with the least amount of solar gain.

The inverse is for cooling - we want the most extreme condition, which is cooling with the most amount of solar gain.